src/MICore/Transports/PipeTransport.cs

```diff
@@ -92,7 +92,7 @@ protected virtual void InitProcess(Process proc, out StreamReader stdout, out St
                 _debuggerPid = _process.Id;
                 stdout = _process.StandardOutput;
                 // Creating a new stream writer to set encoding to UTF-8 with UTF8Identifier as false. This prevents sending Byte Order Mask within the stream.
-                stdin = new StreamWriter(_process.StandardInput.BaseStream, new UTF8Encoding(encoderShouldEmitUTF8Identifier: false), 1024 /* note: this is the default buffer size in the BCL */, leaveOpen: true);
+                stdin = new StreamWriter(_process.StandardInput.BaseStream, new UTF8Encoding(encoderShouldEmitUTF8Identifier: false), STREAM_BUFFER_SIZE, leaveOpen: true);
```

Comment:
Can you make two tweaks here:
1. Just declare the constant here, as it is the only place you should need to use it.
2. Add a comment that we are using a slightly larger than normal buffer as lldb-mi seems to have a bug where commands that exceed the buffer are truncated